pkg/install/resources.go

```diff
@@ -56,13 +58,22 @@ func objectMeta(namespace, name string) metav1.ObjectMeta {
 func ServiceAccount(namespace string) *corev1.ServiceAccount {
 	return &corev1.ServiceAccount{
 		ObjectMeta: objectMeta(namespace, "velero"),
+		TypeMeta: metav1.TypeMeta{
+			Kind:       "ServiceAccount",
+			APIVersion: corev1.SchemeGroupVersion.String(),
+		},
 	}
 }
 
 func ClusterRoleBinding(namespace string) *rbacv1beta1.ClusterRoleBinding {
-	return &rbacv1beta1.ClusterRoleBinding{
+	crb := &rbacv1beta1.ClusterRoleBinding{
 		ObjectMeta: metav1.ObjectMeta{
```

Comment:
you can use the existing `objectMeta` helper func here if you want

Reply:
Does passing in `""` for a namespace make it cluster-scoped? I think that's why it had been left of previously.

Reply:
yeah, it should